.github/workflows/nightly-coordinator-e2e.yml

```diff
@@ -0,0 +1,139 @@
+name: Nightly Ceremony
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+
+env:
+  NODE_OPTIONS: "--max-old-space-size=8192"
+  TTL: ${{ vars.COORDINATOR_TTL }}
+  LIMIT: ${{ vars.COORDINATOR_LIMIT }}
+  COORDINATOR_PUBLIC_KEY_PATH: ${{ vars.COORDINATOR_PUBLIC_KEY_PATH }}
+  COORDINATOR_PRIVATE_KEY_PATH: ${{ vars.COORDINATOR_PRIVATE_KEY_PATH }}
+  COORDINATOR_MACI_PRIVATE_KEY: ${{ secrets.COORDINATOR_MACI_PRIVATE_KEY }}
+  COORDINATOR_TALLY_ZKEY_NAME: ${{ vars.COORDINATOR_TALLY_ZKEY_NAME }}
+  COORDINATOR_MESSAGE_PROCESS_ZKEY_NAME: ${{ vars.COORDINATOR_MESSAGE_PROCESS_ZKEY_NAME }}
+  COORDINATOR_POLL_JOINING_ZKEY_NAME: ${{ vars.COORDINATOR_POLL_JOINING_ZKEY_NAME }}
+  COORDINATOR_POLL_JOINED_ZKEY_NAME: ${{ vars.COORDINATOR_POLL_JOINED_ZKEY_NAME }}
+  COORDINATOR_RAPIDSNARK_EXE: ${{ vars.COORDINATOR_RAPIDSNARK_EXE }}
+  COORDINATOR_ZKEY_PATH: ${{ vars.COORDINATOR_ZKEY_PATH }}
+  COORDINATOR_RPC_URL: ${{ secrets.COORDINATOR_RPC_URL }}
+  COORDINATOR_ADDRESSES: ${{ vars.COORDINATOR_ADDRESSES }}
+  COORDINATOR_ALLOWED_ORIGINS: ${{ vars.COORDINATOR_ALLOWED_ORIGINS }}
+  COORDINATOR_REDIS_HOST: ${{ vars.COORDINATOR_REDIS_HOST }}
+  COORDINATOR_REDIS_PORT: ${{ vars.COORDINATOR_REDIS_PORT }}
+  COORDINATOR_PORT: ${{ vars.COORDINATOR_PORT }}
+  SUBGRAPH_NAME: ${{ vars.COORDINATOR_SUBGRAPH_NAME }}
+  SUBGRAPH_PROVIDER_URL: ${{ vars.COORDINATOR_SUBGRAPH_PROVIDER_URL }}
+  SUBGRAPH_DEPLOY_KEY: ${{ secrets.COORDINATOR_SUBGRAPH_DEPLOY_KEY }}
+  SUBGRAPH_FOLDER: ${{ vars.COORDINATOR_SUBGRAPH_FOLDER }}
+  ZERODEV_BUNDLER_RPC_OP_SEPOLIA: ${{ secrets.COORDINATOR_ZERODEV_BUNDLER_RPC_OP_SEPOLIA }}
+  TEST_PRIVATE_KEY: ${{ secrets.COORDINATOR_TEST_PRIVATE_KEY }}
+  PRIVATE_KEY: ${{ secrets.COORDINATOR_PRIVATE_KEY }}
```

Comment:
I get the following warning locally on this line: `Context access might be invalid: COORDINATOR_PRIVATE_KEY`

Should this be `PRIVATE_KEY` or `COORDINATOR_MACI_PRIVATE_KEY`?

Reply:
I think it makes sense to remove `TEST_PRIVATE_KEY` env and use `PRIVATE_KEY` instead. They do the same thing.